stylesheets/page-content.scss

```diff
@@ -17,4 +17,15 @@
     .btn-primary {
         width: 100%;
     }
+}
+
+.centered-light-heading {
+    font-size: 16px;
+    color: #aaa;
+    text-align: center;
+    margin: 5px 0 5px !important;
+}
+
+.spaced-headings h2 {
+    margin-top: 35px;
```

Comment:
I wouldn't mind even more space here. 100px looks good to me. You will also find an issue with the floating screenshots when you do this.